admin/Entwicklungsumgebungen/Coding-Guidlines/index.md

```diff
@@ -430,14 +411,16 @@ Wenn der Propertyname und der Wert denselben Namen haben, verwende die Kurzschre
 **Richtig:**
 ```vue
 <template>
-  <card-component :title></card-component> <!-- Kurzschreibweise, da Property- und Wertname "title" übereinstimmen -->
+  <!-- Kurzschreibweise, da Property- und Wertname "title" übereinstimmen -->
+  <card-component :title></card-component>
 </template>
 ```
 
 **Falsch:**
 ```vue
 <template>
-  <card-component :title="title"></card-component> <!-- Hier ist die explizite Bindung redundant -->
+  <!-- Hier ist die explizite Bindung redundant -->
+  <card-component :title="title"></card-component>
```

Comment:
Danke, hier sehe ich eine weitere Regel, Inhaltlose Tags sollten selbstschließend sein, also 
```html
<card-component :title="title" />
```
Das ist aber auch so per eslint eingestellt. Kann, muss aber nicht in die Doku, das wird eslint bemängeln.

Reply:
Da der Linter das automatisch  korrigiert, würde ich die Regel nicht dokumentieren. Dann müssten wir alle ESLint Regeln dokumentieren und das würde das Dokument unnötig aufblähen